qiling/loader/utils.py

```diff
@@ -43,12 +43,11 @@ def getident():
         osabi = ident[0x7]
         e_machine = ident[0x12:0x14]
 
-        if osabi == 0x11 or osabi == 0x03 or osabi == 0x0:
-            ostype = QL_OS.LINUX
-        elif osabi == 0x09:
+        if osabi == 0x09:
             ostype = QL_OS.FREEBSD
         else:
-            ostype = None
+        #if osabi == 0x11 or osabi == 0x03 or osabi == 0x0:
```

Comment:
what is the osabi value did you see ? maybe we can add into const or somewhere

Reply:
I got a osabi value 0x78 from a malware binary (Mirai), and I found this happends on multiple binary samples of ARM based Dlink Linux.

Reply:
I could send you guys those specific smaples if needed.

Reply:
Will you be able to put a additional info on the comment out section by adding 0x97 is needed.

Reply:
Or another option is make it osabi in () i guess its better.

Reply:
I will merge and comment